app-backup/holland-backup-example/holland-backup-example-1.0.14.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1
+
+DESCRIPTION="Holland Example Plugin"
+HOMEPAGE="http://www.hollandbackup.org/"
+
+MY_P="${P%%-*}-${P##*-}"
```

Comment:
Please remove the empty lines surrounding this line, too much whitespace makes the ebuild unreadable. Also, this substitution is horribly unreadable. Please use appropriate `PN` and `PV` variables instead of reinventing the wheel. That said, saying `holland` literally would probably be even more readable.